datadog_lambda/tracing.py

```diff
@@ -357,8 +358,10 @@ def extract_context_from_kinesis_event(event, lambda_context):
 
 
 def _deterministic_sha256_hash(s: str, part: str) -> (int, int):
-    sha256_hash = hashlib.sha256(s.encode()).hexdigest()
+    return _sha256_to_binary_part(hashlib.sha256(s.encode()).hexdigest(), part)
```

Comment:
Split this up with a helper so we have an entrypoint for our `sha256_hash` from the `x-datadog-trace-id-hash` and `x-datadog-parent-id-hash`

Reply:
nice!